lib/rules/explicit-createref-type.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const explicitCreateRefTypeMessage = `React.createRef() class properties should be explicitly typed.`;
```

Comment:
What do you think of specifying that the type should be: `{current: null | React.ElementRef<typeof Component>}`?